src/netaddress.cpp

```diff
@@ -150,7 +150,7 @@ void CNetAddr::SetIP(const CNetAddr& ipIn)
     m_addr = ipIn.m_addr;
 }
 
-void CNetAddr::SetLegacyIPv6(Span<const uint8_t> ipv6)
+void CNetAddr::SetLegacyIPv6(const Span<const uint8_t>& ipv6)
```

Comment:
I'm not sure this is worth it. `span` is an extremely light object (a pointer+size), I think it's normally copied by-value because doing an extra dereference on access would be less efficient.